.vscode/extensions.json

```
@@ -1,7 +1,12 @@
 {
-  "recommendations": ["biomejs.biome", "chenglou92.rescript-vscode"],
+  "recommendations": [
+    "biomejs.biome", 
+    "chenglou92.rescript-vscode",    
+    "ocamllabs.ocaml-platform",
+    "rust-lang.rust-analyzer"
+  ],
   "unwantedRecommendations": [
     "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode"
+    "esbenp.prettier-vscode",
```

This is not valid JSON with the trailing comma.

```suggestion
    "esbenp.prettier-vscode"
```